feat(app_home): add Slack community link to home page

PranavKeshav24 opened the pull request:
## Describe your changes

Addresses Issue #171 
Added a “Join Slack Community” button to the App Home view so users can easily access our public Slack workspace. The new button appears as an accessory on a dedicated section in the onboarding flow.

## How has this been tested?

Tested the functionality on the test workspace. The 'Join' button takes the user to the slack page to join the community.

## Screenshots (if appropriate):

![image](https://github.com/user-attachments/assets/274f79c5-9f2e-4c33-88e9-f04df1872581)

PranavKeshav24 commented on src/slack/views/app_home.ts:
```diff
@@ -460,6 +460,16 @@ export const getOnboardingView = (): BlockBuilder[] => {
     }),
+    Blocks.Divider(),
+    Blocks.Section({
+      text: `${Md.emoji('handshake')} *Join Our Slack Community*`
```
The suggested change is not as per the requirements in the issue #171

lalitindoria commented:
Please move the Slack community section towards the end. We should show some indication that if people need help they can join the slack community.

PranavKeshav24 commented:
I've moved the Slack community section to the end and added a note indicating the users to join if they need help. Let me know if you'd like any further changes!

batflarrow commented on src/slack/views/app_home.ts:
```diff
@@ -464,3 +464,18 @@ export const getOnboardingView = (): BlockBuilder[] => {
+  return [
+    Blocks.Section({
+      text: `${Md.emoji('handshake')} *Need help or want to connect with others?* Join our Slack community!`
+    }).accessory(
```
```suggestion
      text: `${Md.emoji('busts_in_silhouette')} Join our Slack community to get help and connect with other Quix users.`
```

batflarrow commented on src/slack/views/app_home.ts:
```diff
@@ -464,3 +464,18 @@ export const getOnboardingView = (): BlockBuilder[] => {
+export const getCommunityLinkView = (): BlockBuilder[] => {
+  return [
+    Blocks.Section({
+      text: `${Md.emoji('handshake')} *Need help or want to connect with others?* Join our Slack community!`
```
```suggestion
       text: `${Md.emoji('busts_in_silhouette')} Join our Slack community to get help and connect with other Quix users`  

```

PranavKeshav24 replied:
I've updated the text as suggested in the latest commit.

batflarrow commented on src/slack/views/app_home.ts:
```diff
@@ -464,3 +464,18 @@ export const getOnboardingView = (): BlockBuilder[] => {
+      text: `${Md.emoji('handshake')} *Need help or want to connect with others?* Join our Slack community!`
+    }).accessory(
+      Elements.Button({
+        text: 'Join',
```
```suggestion
         text: `${Md.emoji('slack')}  Join Community`,
```
